static/app/components/core/button/types.tsx

```diff
@@ -0,0 +1,57 @@
+import type {TooltipProps} from 'sentry/components/core//tooltip';
```

Comment:
weird that this works and has no error, but I think it should be:

```suggestion
import type {TooltipProps} from 'sentry/components/core/tooltip';
```